src/execution.ts

```diff
@@ -775,7 +789,7 @@ function unpromisify(
         value
             .then(
                 (res: any) => cb(res, null),
-                (err: Error) => (err != null ? cb(null, err) : cb(null, new Error("")))
+                (err: Error) => (err != null ? cb(null, err) : cb(null, new (CustomGraphQLError as any)("")))
```

Comment:
You can do something like:
```
class CustomGraphQLError {
  constructor(message) {
    this.message = message;
    this.name = "CustomGraphQLError";
    return Object.setPrototypeOf(this, Error.prototype);
  }
  toString() {
    return `${this.name}: ${this.message}`;
  }
}
```
then:

```
const e = new CustomGraphQLError("Some error");
console.log(e instanceof Error) // true
```
The performance penalty is minimal.

Reply:
Use `GraphqlJitError`. or `GraphQLError`